wooey/models/favorite.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from django.conf import settings
+from ..django_compat import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+
+class Favorite(models.Model):
+
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, related_name='favorites', blank=True, null=True)
+
+    content_type = models.ForeignKey(ContentType)
+    object_id = models.PositiveIntegerField()
+    content_object = GenericForeignKey('content_type', 'object_id')
```

Comment:
I'm good with the 2 query method. I forgot that approach. I think a custom manager would work best for that to assign the model attribute to the Favorite model after returning the query. So we can just work with something like:

[i.favorite for i in Favorite.objects.filter(user=user)]

and be able to forget any of the behind the scenes.